Documentation/mgcb.md

```diff
@@ -110,9 +110,9 @@ Note all defined processor parameters are cleared when the `/processor` is set.
 
 ### Build Content File
 ```
-/build:<content_filepath>
+/build:<content_filepath>;*<destination_filepath>
```

Comment:
Let's just write this twice to make the syntax clear (`*` does not suggest optional to me):
```
/build:<content_filepath>
/build:<content_filepath>;<destination_filepath>
```

Reply:
Done.